src/nav2_tasks/include/nav2_tasks/TaskClient.hpp

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2018 Intel Corporation
```

Comment:
I think the TaskClient.hpp file was accidentally added back in when the last commit was pushed. It was previously renamed to task_client.hpp, right? Now they're both in.